plugins/jira/plugin.spec.yaml

```
@@ -574,3 +574,39 @@ triggers:
         type: issue
         description: New issue
         required: false
+  updated_issue:
+    title: Updated Issue
+    description: Trigger which indicates that an issue has been updated or a new one has been created
+    input:
+      projects:
+        title: Projects
+        description: List of Project IDs or names
+        type: "[]string"
+        required: false
+        example: TEST
+      jql:
+        title: JQL
+        type: string
+        description: JQL search string to use
+        required: false
+        example: "reporter='Example User'"
+      get_attachments:
+        title: Get Attachments
+        description: Get attachments from issue
+        required: false
+        type: boolean
+        default: false
+        example: true
+      poll_timeout:
+        title: Poll Timeout
+        description: Timeout between next poll, default 60
+        required: false
+        type: integer
+        default: 60
+        example: 60
```

```suggestion
      interval:
        title: Interval
        description: Interval between next poll in seconds, default is 60 seconds
        required: false
        type: integer
        default: 60
        example: 60
```

Let's go with `interval` based on https://docs.rapid7.com/insightconnect/style-guide/#inputs